packages/react-native-audio-api/ios/audioapi/ios/system/AudioSessionManager.h

```diff
@@ -6,6 +6,7 @@
 @interface AudioSessionManager : NSObject
 
 @property (nonatomic, weak) AVAudioSession *audioSession;
+@property (nonatomic, weak) AVAudioApplication *audioApplication;
```

Comment:
AVAudioApplication should be also used only on iOS > 17

https://developer.apple.com/documentation/avfaudio/avaudioapplication?language=objc

You can use this type of checks:
https://developer.apple.com/documentation/swift/marking-api-availability-in-objective-c

also for simplicity I wouldn't store AVAudioApplication as a class property